storage/engine/mvcc.go

```diff
@@ -49,6 +50,36 @@ var (
 	NilKey = MVCCKey{}
 )
 
+// AccountForSelf adjusts ms to account for the predicted impact it will have on
+// the values that it records when the structure is initially stored. Specifically,
+// MVCCStats is stored on the RangeStats key, which means that its creation will
+// have an impact on system-local data size and key count.
+func AccountForSelf(ms *enginepb.MVCCStats, rangeID roachpb.RangeID) error {
```

Comment:
Yeah, awkward. Not sure how to make this look better.

On Sat, Jun 11, 2016 at 4:21 PM Tamir Duberstein <notifications@github.com>
wrote:

> In storage/engine/mvcc.go
> <https://github.com/cockroachdb/cockroach/pull/7166#discussion_r66711224>:
>
> > @@ -49,6 +50,36 @@ var (
> >  	NilKey = MVCCKey{}
> >  )
> >
> > +// AccountForSelf adjusts ms to account for the predicted impact it will have on
> > +// the values that it records when the structure is initially stored. Specifically,
> > +// MVCCStats is stored on the RangeStats key, which means that its creation will
> > +// have an impact on system-local data size and key count.
> > +func AccountForSelf(ms *enginepb.MVCCStats, rangeID roachpb.RangeID) error {
>
> can't this go back to being a method now?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7166/files/c986fa5e308346a0050cf4d2e76e02ccfdeb7527..a0846d3e36776522759174b0bff8589c0f900a82#r66711224>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE135F-v-Io0DeTJJCuLnD--oJSPlX-Hks5qKxixgaJpZM4IzeMe>
> .
>
-- 

-- Tobias